app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  before_action :redirect_users_to_home, only: [:new, :create]
+
+  def create
+    user = User.find_by_email(params[:email])
```

Comment:
Refrain from `find_by_xxx` that is inefficient and goes through undefined method. Instead use `find_by(email: params[:email])`.